### ISSUE TITLE
Missing street property

### ISSUE THREAD

**Issue by otbutz** [komoot/photon#402]
A query for `Regensburg` returns `Regensburg` as the value for `city` and `name` properties:
http://photon.komoot.de/api?q=Regensburg
```json
{
	"features": [{
		"geometry": {
			"coordinates": [12.0974869, 49.0195333],
			"type": "Point"
		},
		"type": "Feature",
		"properties": {
			"osm_id": 240120926,
			"osm_type": "N",
			"country": "Germany",
			"osm_key": "place",
			"city": "Regensburg",
			"osm_value": "city",
			"postcode": "93047",
			"name": "Regensburg",
			"state": "Bavaria"
		}
	}, .... ],
	"type": "FeatureCollection"
}
```
Querying for `Talstraße 70188 Stuttgart` returns a way result without the `street` property:
http://photon.komoot.de/api?q=Talstra%C3%9Fe%2070188%20Stuttgart
```json
{
	"features": [ .... , {
		"geometry": {
			"coordinates": [9.2176295, 48.7865909],
			"type": "Point"
		},
		"type": "Feature",
		"properties": {
			"osm_id": 349219890,
			"osm_type": "W",
			"extent": [9.2175014, 48.7865909, 9.2176295, 48.7865231],
			"country": "Germany",
			"osm_key": "highway",
			"city": "Stuttgart",
			"osm_value": "secondary",
			"postcode": "70188",
			"name": "Talstraße",
			"state": "Baden-Württemberg"
		}
	}],
	"type": "FeatureCollection"
}
```

Shouldn't the `street` tag be set with value `Talstraße` here?

**Comment by simonpoole** [comment]
I would argue in a way that the first result is actually wrong by including the city value :-), but it depends a bit on what the intended semantics of the result are (partial address vs simply the location and name of the object in question).

**Comment by otbutz** [comment]
> but it depends a bit on what the intended semantics of the result are (partial address vs simply the location and name of the object in question).

I'd argue that a query for `Talstraße 117, 70188 Stuttgart` should at least return a partial result if the housenumber is not part of the dataset. Otherwise the result parsing will be cumbersome.

**Comment by lonvia** [comment]
I agree with @simonpoole that the result for Regensburg is the wrong one here. This gets fixed with #457.

We can't really give a different result for `Talstraße 117, 70188 Stuttgart` and `Talstraße, 70188 Stuttgart` because we'd have to figure out first, that the first query was for a house number and the second for a street. And that's a lot harder than you think.

**Comment by lonvia** [comment]
Regensburg now shows up correctly in the latest graphopper extracts. I expect that komoot updates their instance soon with the latest data.

Regarding the `Talstrasse` example: what is really needed is the information that Photon does not have the exact housenumber and has just returned the street instead. This is best done by implementing the `type` field. I've opened #470 for that. Closing here.
